pkg/build/build.go

```diff
@@ -866,3 +868,26 @@ func (ctx *Context) WorkspaceCmd(args ...string) (*exec.Cmd, error) {
 
 	return cmd, nil
 }
+
+// BuildFlavor determines if a build context uses glibc or musl, it returns
+// "gnu" for GNU systems, and "musl" for musl systems.
```

Comment:
Not blocking: Might be worth a hacky Go enum for this eventually, if there are only a few strings we expect this to ever be.